app/survey/survey_copy.py

```diff
@@ -0,0 +1,686 @@
+import copy
```

Comment:
you copied the survey.py file ? The point of version control is that you don't need to do this :)